### PR TITLE
Fixes on Edit button, there is + sign overlayed over letter E #5388

### LINKED ISSUE #5388: [Bug]: on `Edit` button, there is `+` sign overlayed over letter `E`
### Summary

In `Step 1/4: Media Details` part of the upload, the `EDIT IMAGE` button shown at bottom left has `+` sign overlayed over letter `E`.

### Steps to reproduce

1. open the app
2. click on `+` button and choose gallery
3. select some picture

### Expected behaviour

window that opens should have normal text `EDIT IMAGE` without any `+` sign overlayed (or, if there is some extra functionality being offered by that `+` sign, it should be separate and clickable). 

### Actual behaviour

`EDIT IMAGE` is shown with `+` over letter `E`, making it unreadable

### Device name

Huawei P30Pro

### Android version

Android 10 (EMUI 12)

### Commons app version

4.2.1 ~14b6c455b (latest f-droid)

### Device logs

_No response_

### Screen-shots

Screenshot_20231117_134256_fr free nrw commons

### Would you like to work on the issue?

None

### PR DESCRIPTION
**Pull request by kanahia1**
**Description (required)**

Fixes #5388

What changes did you make and why?

TV - 2

**Tests performed (required)**
Tested betaDebug on Samsung S21 FE with API level 33.

**Screenshots (for UI changes only)**

`Upload Activity`
https://github.com/commons-app/apps-android-commons/assets/114223204/17a95ff8-0670-4fcc-b870-cef1d9163f30

`Description Edit Activity`
https://github.com/commons-app/apps-android-commons/assets/114223204/55ce869e-0753-440f-b2b0-3b498df6110f

### CODE REVIEW

File: app/src/main/java/fr/free/nrw/commons/upload/UploadMediaDetailAdapter.java [hunk 1]
```diff
@@ -557,6 +577,51 @@ public String convertIdeographicSpaceToLatinSpace(String source) {
 
     }
 
+    private void updateAddButtonVisibility() {
```
Comment (nicolas-raoul) [inline app/src/main/java/fr/free/nrw/commons/upload/UploadMediaDetailAdapter.java:580]: Could you please add some Javadoc?

File: app/src/main/java/fr/free/nrw/commons/description/DescriptionEditActivity.kt [hunk 2]
```diff
@@ -111,18 +110,17 @@ class DescriptionEditActivity : BaseActivity(), UploadMediaDetailAdapter.EventLi
     }
 
     override fun onPrimaryCaptionTextChange(isNotEmpty: Boolean) {}
-
-    private fun onBackButtonClicked(view: View) {
-        onBackPressed()
-    }
-
-    private fun onButtonAddDescriptionClicked(view: View) {
+    override fun addLanguage() {
```
Comment (nicolas-raoul) [inline app/src/main/java/fr/free/nrw/commons/description/DescriptionEditActivity.kt:113]: Could you please add some Javadoc?

File: app/src/main/java/fr/free/nrw/commons/upload/mediaDetails/UploadMediaDetailFragment.java [hunk 3]
```diff
@@ -711,6 +710,14 @@ public void onPrimaryCaptionTextChange(boolean isNotEmpty) {
         btnNext.setAlpha(isNotEmpty ? 1.0f : 0.5f);
     }
 
+    @Override
```
Comment (nicolas-raoul) [inline app/src/main/java/fr/free/nrw/commons/upload/mediaDetails/UploadMediaDetailFragment.java:713]: Could you please add some Javadoc?

**Review by nicolas-raoul — CHANGES_REQUESTED** [review]
It works great!
The code looks good, apart from the Javadoc :-)

File: app/src/main/java/fr/free/nrw/commons/upload/UploadMediaDetailAdapter.java (hunk 1 above)
Comment (kanahia1) [reply app/src/main/java/fr/free/nrw/commons/upload/UploadMediaDetailAdapter.java:584]: Sure ! and Thanks for reporting, I have added javadoc in newer commit https://github.com/commons-app/apps-android-commons/pull/5464/commits/f425eadf17b6d18f861dc95c66a7d71f6085a2c7

**Review by nicolas-raoul — APPROVED** [review]

**Pull request merged**